docker-compose.test.yml

```diff
@@ -10,9 +10,14 @@ services:
         NODE_ENV: 'development'
     environment:
       NODE_APP_INSTANCE: 'docker'
-    command: 'npm test'
+    command: >
+      sh -c "mkdir .nyc_output && npx nyc npm test 
+      && npx nyc report --reporter=cobertura --reporter=text-summary > coverage-summary.txt 
+      && cp coverage/cobertura-coverage.xml /app/output/coverage.cobertura.xml"
     depends_on:
       - db
+    volumes:
+      - ./output_directory:/app/output
```

Comment:
@Patel-world  - Can we change `output_directory` to `output` ?

Reply:
Ref - https://github.com/jalantechnologies/github-ci/pull/17/files#r1284058136

Can we change coverage file name to `coverage.xml` ?